scripts/build/termux_step_make.sh

```diff
@@ -25,3 +25,7 @@ termux_step_make() {
 			${TERMUX_PKG_EXTRA_CONFIGURE_ARGS:+ $TERMUX_PKG_EXTRA_CONFIGURE_ARGS}
 	fi
 }
+
+termux_step_make32() {
+	termux_step_make
```

Comment:
Where are termux_step_make32 and termux_step_make_install32 called? Why wrap a straightforward termux_step_make and termux_step_make_install?

Reply:
> Where are termux_step_make32 and termux_step_make_install32 called?

These functions are run via the `termux_run_cross_function` function - https://github.com/termux/termux-packages/blob/adding-glibc32-prefix/build-package.sh#L439

> Why wrap a straightforward termux_step_make and termux_step_make_install?

This allows you to avoid configuring these functions in packages, provided that the build steps of a regular package and a 32-bit package are not very different and it is possible to configure their builds through a single function. You can look at the example build.sh of a glibc package that has 32-bit compilation enabled to better understand the use of these functions - https://github.com/termux-pacman/glibc-packages/blob/glibc/gpkg/glibc/build.sh